osi_referenceline.proto

```diff
@@ -202,6 +204,14 @@ message ReferenceLine
         // S position on the reference line
         //
         optional double s_position = 2;
+
+        // Yaw angle of the T axis in the world coordinate system
+        //
+        // When converting from formats like OpenDRIVE, the yaw angle is equal to
```

Comment:
I guess it doesn't matter whether the normal goes to the left or to the right - correct?

That also means that "there is no I because two consecutive t_axis_yaw are the same" can also happen if the two angles differ by 180 degree (or PI, since this is stored in radians). Correct?

Reply:
Correct